metron-sensors/bro-plugin-kafka/README.md

```diff
@@ -0,0 +1,160 @@
+Bro Logging Output to Kafka
+===========================
+
+A Bro log writer that sends logging output to Kafka.  This provides a convenient
+means for tools in the Hadoop ecosystem, such as Storm, Spark, and others, to
+process the data generated by Bro.
+
+Installation
+------------
+
+Install librdkafka (https://github.com/edenhill/librdkafka), a native client
+library for Kafka.  This plugin has been tested against the latest release of
+librdkafka, which at the time of this writing is v0.9.4.  In order to support interacting
+with a kerberized kafka, you will need libsasl2 installed
+
+    # curl -L https://github.com/edenhill/librdkafka/archive/v0.9.4.tar.gz | tar xvz
+    # cd librdkafka-0.9.4/
+    # ./configure --enable-sasl=true
```

Comment:
Small typo.  I think it needs to be `./configure --enable-sasl`.  Passing true as an argument causes it to be unhappy.